apps/spruce/src/hooks/useFirstImage.ts

```diff
@@ -0,0 +1,14 @@
+import { useQuery } from "@apollo/client";
+import { ImagesQuery, ImagesQueryVariables } from "gql/generated/types";
+import { IMAGES } from "gql/queries";
+
+/**
+ * `useFirstImage` returns the alphabetically first image from Evergreen's list of images.
+ * This can be used to generate a general link to images settings.
+ * @returns an object containing the distro ID (string) and loading state (boolean)
```

Comment:
Comment needs update (not a distro ID)

Reply:
Could we maybe write a test for this hook as well? I think it might be good practice since you'll learn how to mock GraphQL queries in the tests!